client/src/components/Questions/Questions.jsx

```diff
@@ -34,7 +34,8 @@ class Questions extends React.Component {
   getQuestions(product) {
     axios({
       method: 'get',
-      url: url.resolve('http://localhost:3003/questions/', product),
+      // url: url.resolve('http://localhost:3003/questions/', product),
+      url: url.resolve('/questions/', product),
```

Comment:
Ive not seen this line of code before. Wonder what resolve does😁

Reply:
using url.resolve it similar to using path.join for file paths